### PR TITLE
Implement Stages UI

### PR DESCRIPTION
**Pull request by michalsidzej**
Resolves L2B-1581

### CODE REVIEW

File: packages/config/src/layer2s/common/stages/types.ts [hunk 1]
```diff
@@ -20,21 +20,30 @@ export type ChecklistTemplate<T extends StageBlueprint> = {
   }
 }
 
+// TODO: maybe it shouldn't be undefined
+export type Stage = 'Stage 0' | 'Stage 1' | 'Stage 2' | undefined
```
Comment (antooni) [inline packages/config/src/layer2s/common/stages/types.ts:24]: 'Candidate' ?

File: packages/frontend/src/components/stages/StageDisclaimer.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import cx from 'classnames'
+import React from 'react'
+
+import { InfoIcon } from '../icons'
+
+export function StageDisclaimer({
```
Comment (antooni) [inline packages/frontend/src/components/stages/StageDisclaimer.tsx:6]: use `Callout` component for this

File: packages/frontend/src/components/stages/StageTooltip.tsx [hunk 3]
```diff
@@ -1,34 +1,83 @@
...
-            {item.missing.requirements.map((requirement, i) => (
-              <li className="w-60" key={i}>
-                ❌ {requirement}
-              </li>
-            ))}
-          </ul>
-        </div>
+      {item.stage === 'UnderReview' ? (
+        <>
+          Projects under review might present uncompleted information & data.
```
Comment (antooni) [inline packages/frontend/src/components/stages/StageTooltip.tsx:25]: this text seems a bit too long, but it is a nitpick

**Review by antooni — CHANGES_REQUESTED** [review]
kudos, gj

please resolve comments before merging

File: packages/config/src/layer2s/common/stages/types.ts (hunk 1 above)
Comment (michalsidzej) [reply packages/config/src/layer2s/common/stages/types.ts:24]: Yeah, I was thinking about that, but we do not have any designs ready for this case. We could ignore it for now IMO

File: packages/frontend/src/components/stages/StageTooltip.tsx (hunk 3 above)
Comment (michalsidzej) [reply packages/frontend/src/components/stages/StageTooltip.tsx:25]: What do you propose then?

Comment (antooni) [reply packages/frontend/src/components/stages/StageTooltip.tsx:25]: Let's leave it for now, I do not have an idea

**Review by antooni — APPROVED** [review]

**Pull request merged**